charmtools/git_version.py

```
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import os
+import sys
+import json
+import argparse
+from subprocess import check_output, CalledProcessError
+from pkg_resources import parse_version
+
+
+git_cmd = ['git', 'describe', '--tags', '--long']
+version_filename = os.path.join(os.path.dirname(__file__), '..', 'VERSION')
+
+
+def get_version_info():
+    try:
+        version, gitn, gitsha = check_output(git_cmd).strip().rsplit('-', 2)
```

Can we just ignore this error from codacy?